doc/api/cli.md

```diff
@@ -1663,6 +1648,25 @@ Disable the `node-addons` exports condition as well as disable loading
 native addons. When `--no-addons` is specified, calling `process.dlopen` or
 requiring a native C++ addon will fail and throw an exception.
 
+### `--no-async-context-frame`
+
+<!-- YAML
+added: v22.7.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55552
+    description: Marked as stable.
```

Comment:
This is a new flag, does it needed a "changes"?

Reply:
Ah, true. It was a lift-and-shift from the previous docs. I've updated it to look more like a _new_ flag. 😅